model/project_event.go

```diff
@@ -108,12 +108,33 @@ func (p *ProjectChangeEvents) RedactPrivateVars() {
 		if !isChangeEvent {
 			continue
 		}
+		modifiedPrivateVarKeys := findModifiedPrivateVars(changeEvent)
 		changeEvent.After.Vars = *changeEvent.After.Vars.RedactPrivateVars()
 		changeEvent.Before.Vars = *changeEvent.Before.Vars.RedactPrivateVars()
+		// If a private variable has been changed, change the before value of the
```

Comment:
Could we do this for the "after" value instead of the before value? I worry slightly that if the UI shows `{REDACTED} -> ""` it's going to make people think the value isn't redacted anymore.